### PR TITLE
add paraBem

### PR DESCRIPTION
**Pull request by looooo**

### CODE REVIEW

**Comment by looooo** [comment]
I guess this ready @conda-forge/help-c-cpp

File: recipes/paraBEM/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+
+{% set name = "parabem" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "0b2a87a788cc658b85e35b9ed8c1e4c02b1e34a300aad230dee42379dfad36d1" %}
+
+package:
+    name: parabem
```
Comment (chrisburr) [inline recipes/paraBEM/meta.yaml]:
```suggestion
    name: {{ name }}
```

File: recipes/paraBEM/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        - numpy
+        - matplotlib
+        - llvm-openmp  # [osx]
+test:
+    imports:
+        - paraBEM
+
+about:
+  home: https://github.com/booya-at/paraBEM
+  license: GPL3
```
Comment (chrisburr) [inline recipes/paraBEM/meta.yaml]:
```suggestion
  license: GPL-3.0
```

File: recipes/paraBEM/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  license_family: GPL
+  license_file: LICENSE
+  summary: potential flow with boundary elements.
+extra:
```
Comment (chrisburr) [inline recipes/paraBEM/meta.yaml]:
```suggestion

extra:
```

File: recipes/paraBEM/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+test:
+    imports:
+        - paraBEM
+
+about:
+  home: https://github.com/booya-at/paraBEM
+  license: GPL3
+  license_family: GPL
+  license_file: LICENSE
+  summary: potential flow with boundary elements.
```
Comment (chrisburr) [inline recipes/paraBEM/meta.yaml]:
```suggestion
  summary: potential flow with boundary elements.
  doc_url: https://github.com/booya-at/paraBEM/blob/master/doc/tutorial/tutorial.ipynb
  dev_url: https://github.com/booya-at/paraBEM
```

File: recipes/paraBEM/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+        - pybind11
+        - eigen
+        - openmpi  # [linux]
+        - llvm-openmp  # [osx]
+    run:
+        - python
+        - numpy
+        - matplotlib
+        - llvm-openmp  # [osx]
+test:
```
Comment (chrisburr) [inline recipes/paraBEM/meta.yaml]:
```suggestion

test:
```

File: recipes/paraBEM/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+{% set name = "parabem" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "0b2a87a788cc658b85e35b9ed8c1e4c02b1e34a300aad230dee42379dfad36d1" %}
+
+package:
+    name: parabem
+    version: {{ version }}
+
+source:
+    fn: parabem.tar.gz
```
Comment (chrisburr) [inline recipes/paraBEM/meta.yaml]: This line isn't needed.
```suggestion
```

**Comment by looooo** [comment]
@chrisburr thanks for the review

**Pull request merged**